ds/list/list.go

```diff
@@ -439,3 +439,19 @@ func (l *List) IsEmpty(key string) (bool, error) {
 	}
 	return true, nil
 }
+
+func (l *List) GetListTTL(key string) (uint32, error) {
+	if l.IsExpire(key) {
+		return 0, ErrListNotFound
+	}
+	ttl := l.TTL[key]
+	timestamp := l.TimeStamp[key]
```

Comment:
Seems the TTL implementation consists of two times in the codebase. It is not very reasonable to me, maybe I lack some context here. So would you or @xujiajun like to kindly have a discussion talk about the implementation of the TTL feature with me when you have time? Thanks in advance.